### DISCUSSION TITLE
MaterialFX with Maven

### DISCUSSION THREAD

**Discussion by DevDarkWolf** [palexdev/MaterialFX#154] [Archived]
Hello

Several days ago i found this proyect and i and think this is amazing 

Today i decide used it in a litle proyect, but when i tried to use it with maven, the IDE showed me the next error: 

```
Error initializing QuantumRenderer: no suitable pipeline found
java.lang.RuntimeException: java.lang.RuntimeException: Error initializing QuantumRenderer: no suitable pipeline found
```

And i would like to know if somebody have some idea of why happend the error.

PD: I am testing it in a clean proyect but when i find a solution the idea is used MaterialFX in my proyect

PD2: Sorry for my bad english XD

**Comment by rileyhawk1417** [comment]
Did you try and install virtualizedfx?

**Reply by DevDarkWolf** [reply]
No yet, is necessary?

**Comment by DevDarkWolf** [comment]
Ok, today finally i was able to use the library with help of a friend

The error is in the version that is being used

The problem is that the MaterialFX dependencies in the latest version are classified for linux which causes the above error.
In the version 11.12.0 the dependencies are classified for windows, and the solucion for the insue is basically use this version and not the last.


**11.13.3**
![image](https://user-images.githubusercontent.com/55326999/157937148-976b0b51-03f2-4dc6-bfe5-ef0525524367.png)

**11.12.0**
![image](https://user-images.githubusercontent.com/55326999/157937544-1b476cdf-2f56-4750-aacd-d8a6bf5c3d0e.png)

**Reply by palexdev** [reply]
Happy to hear that!
Unfortunately I know almost nothing about Maven (I kind of hate it to be honest), so I don't what a classifier is
I mean, I understand its purpose but on Gradle there's no such thing, it's much more straightforward

**Reply by JersonVega** [reply]
Hi to all! I had this same issue and this solution worked for me. But, there's a way to remove this classifiers from the **pom.xml** of MaterialFX proyect itself? It seems that this problem cames directly from maven repository.

**Reply by palexdev** [reply]
Honestly I don't know, like I said I don't know how Maven works
But if anyone has a solution for that I won't hesitate fixing it

**Reply by DevDarkWolf** [reply]
Yes i understand
Fortunately exists a solution for the error, maybe exists a way to solve in  maven

**Reply by JersonVega** [reply]
@palexdev I'm willing to help. But how?

(Update)
I've been doing some researching, soon i'll try to post the solution.
